packages/nc-gui/components/smartsheet/Gallery.vue

```diff
@@ -59,6 +59,8 @@ provide(IsFormInj, ref(false))
 provide(IsGalleryInj, ref(true))
 provide(IsGridInj, ref(false))
 
+provide(RowHeightInj, ref(1 as const))
```

Comment:
is `as const` required` here?

Reply:
![image](https://github.com/nocodb/nocodb/assets/64391274/5fd30d8e-6a0a-4311-96c1-e1a56884aff1)
Yes. The possible values are 1, 2, 4 and 6